web/gds-admin-ui/src/utils/roundUptoTwo.js

```diff
@@ -0,0 +1,8 @@
+
+export default function roundUpToTwo(num) {
+    if (isNaN(num)) {
+        throw new TypeError(`"${num}" is not a number`)
```

Comment:
when you return a Typerror here ,it mean you should catch it and handle it where it called. But it seem like you can that function on Object.Entries method. what will happen when it's throw an error  for a string value ?